impf/browser.py

```diff
@@ -322,14 +322,23 @@ def search_appointments(self) -> bool:
         title = self.wait.until(EC.presence_of_element_located((By.XPATH, '//h1')))
         assert title.text == 'Onlinebuchung für Ihre Corona-Schutzimpfung'
 
+        try:
+            close = self.driver.find_elements_by_xpath('//button[contains(text(), "Abbrechen")]')[-1]
+            close.click()
+            sleep(1)
+        except NoSuchElementException:
```

Comment:
This won't be a NoSuchElementException, but a "NotClickableException" (I forgot the corret name).

Also, the first time `search_appointments` is executed, we expect the exception to be raised as the popup hasn't been opened yet. So maybe this should be of log level DEBUG?

Reply:
You're right; depends on whether or not the popup window is already loaded. Pushed a quick commit.